### PR TITLE
Readme

### LINKED ISSUE #64: README  - Rails
- Description
- DB Schema
- Architecture Diagram
- Link to Miro
- Discuss Optimization

### PR DESCRIPTION
**Pull request by Jonathan-M-Wilson**
# Description
Created a rough draft of our project readme. 

# Closes issue(s)
Closes #64 

# Checklist
- [x] I have updated the Readme

# Other comments
I commented in a to-do list at the top of the Readme for when we get closer to having all the finals touches merged in and we are ready for a demo video and screenshots. Please feel free to add, remove, or update this readme as you see fit!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,222 @@
...
+    <a href="https://github.com/Shelf-Share-BEE-Mod3/shelf-share-rails/issues">Request Feature</a>
+  </p>
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [About the Project](#about-the-project)
```
Comment (Strompy) [inline README.md:41]: Great idea

File: README.md [hunk 2]
```diff
@@ -1,7 +1,222 @@
...
+## About The Project
+
+[![about-screenshot][about-screenshot]](https://example.com)
+
+ShelfShare wants to help you save money by allowing you to borrow and lend out books you own to your friends. We put those dusty books in your collection to use!
+
+Here's why:
+* Most of us are guilty of buying that book we "have to have" just to read it once and stash it on a shelf somewhere.
+* Save money! Shelf share is a free service. Once you find a book you are interested in on a friend's shelf, request to borrow the book, pay for shipping, and you are ready to go!
+* Convenience. ShelfShare is simplistic in design, no walking around the library trying to find a book. Simply look through the books that your friends own and request a book! Easy!  :smile:
```
Comment (Strompy) [inline README.md:65]: Way to turn our design limitations into a selling point!

File: README.md [hunk 3]
```diff
@@ -1,7 +1,222 @@
...
+* [Sinatra](https://github.com/sinatra/sinatra)
+* [Rails](https://github.com/rails/rails)
+* [Ruby](https://github.com/ruby/ruby)
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To use ShelfShare locally, you can fork or clone [these](https://github.com/Shelf-Share-BEE-Mod3) two repos.
+The **ShelfShare Web App** is the application itself, while **BookBuddy** is the micro-service we will be using to find the book information we need to add books to our shelf!
```
Comment (Strompy) [inline README.md:81]: This may be a good spot to link to the BookBuddy repo

File: README.md [hunk 4]
```diff
@@ -1,7 +1,222 @@
...
+
+DISCLAIMER:
The ShelfShare platform is a proof-of-concept and not likely to be maintained over time. Thus, any address information submitted will remain in our database indefinitely unless you delete it.
 For concerns regarding your privacy, please do not submit any real location information to this website.
+
+
+<!-- CONTACT -->
+## Contact
+
+Phillip S - [![LinkedIn][linkedin-shield]](https://www.linkedin.com/in/phillipstrom/) - [GitHub](https://github.com/Strompy) - [Email](phillip.strom@gmail.com)
```
Comment (Strompy) [inline README.md:188]: Should we have emails in here? 
Ian also mentioned linking the Turing Portfolio pages

**Review by Strompy — COMMENTED** [review]
Looks great! One other thing I remember Ian mentioning was the Travis badge

File: README.md (hunk 3 above)
Comment (Jonathan-M-Wilson) [reply README.md:81]: Book Buddy is located in that link.

**Pull request merged**